src/loaders/ObjectLoader.js

```diff
@@ -471,16 +471,9 @@ THREE.ObjectLoader.prototype = {
 
 					if ( data.focus !== undefined ) object.focus = data.focus;
 					if ( data.zoom !== undefined ) object.zoom = data.zoom;
-
-					if ( data.filmGauge !== undefined ) {
-
-						if ( data.view !== null )
-							object.view = Object.assign( {}, data.view );
-
-						object.filmGauge = data.filmGauge;
-						object.filmOffset = data.filmOffset;
-
-					}
+					if ( data.filmGauge !== undefined ) object.filmGauge = data.filmGauge;
+					if ( data.filmOffset !== undefined ) object.filmOffset = data.filmOffset;
+					if ( data.view !== undefined ) object.view = Object.assign( {}, data.view );
```

Comment:
`null` is the default, so testing for `data.view !== undefined && data.view !== null` or just `data.view != null` will fix it.